bedrock/products/templates/products/vpn/resource-center/base-article.html

```diff
@@ -0,0 +1,86 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{% block article_title %}{% endblock article_title %}{% endblock page_title%}
+
+{% block content %}
+<ol class="mzp-c-breadcrumb">
+  <li class="mzp-c-breadcrumb-title">
+    <a class="mzp-c-breadcrumb-item-link" href="{{ url('products.vpn.resource-center.landing') }}">{{ ftl('vpn-resource-center-title') }}</a>
+  </li>
+  <li class="mzp-c-breadcrumb-item">
+    {{ self.article_title() }}
+  </li>
+</ol>
+<div class="vpn-article-container">
+  <div class="mzp-l-content mzp-t-content-md">
+      <article class="mzp-c-article vpn-article">
+        {# Categories will be used at some point but not yet #}
+        {# <p class="vpn-article-category">{% block article_category %}{{ self.article_category().capitalize() }}{% endblock %}</p> #}
+        <h2>{{ self.article_title() }}</h2>
+        {% block vpn_article %}{% endblock vpn_article %}
+      </article>
+    <div class="vpn-c-vote">
+      <h3>{{ ftl('vpn-resource-center-was-this-article') }}</h3>
+      <div class="vpn-c-vote-btn-container">
+        <button data-cta-type="button" data-cta-text="{% block article_slug %}{% endblock %}" class="vpn-c-vote-btn up">
+          <svg width="42" height="38" fill="currentColor" xmlns="http://www.w3.org/2000/svg">
+            <path d="M36.54 13.104h2.184A3.277 3.277 0 0 1 42 16.38v17.473a3.277 3.277 0 0 1-3.276 3.276H36.54a3.277 3.277 0 0 1-3.276-3.276V16.38a3.277 3.277 0 0 1 3.276-3.276ZM5.38 10.752c-3.669 0-6.261 3.595-5.1 7.076l4.48 13.44a5.375 5.375 0 0 0 5.1 3.677h19.036V15.238L24.19 8.063l-2.264-6.796A1.857 1.857 0 0 0 20.166 0h-.007a4.032 4.032 0 0 0-4.032 4.032v5.914l-.806.806H5.38Z" fill="currentColor"/>
+          </svg>
```

Comment:
From an a11y perspective, these buttons should contain some localized text to indicate "yes" or "no". Right now, I'm not sure how screen reader users would tell what each button does. I'd suggest using `@image-replaced` to visually hide the text, but that might mean moving the `svg` back to CSS.

Also, this may not need to be a blocker (maybe something we could file a followup issue for?). But how would people know they have noted? The only visual indicator is a change in the icon color, which isn't all that useful if you're visually impaired of color blind. Maybe some sort of visual message could be a nice UX improvement?

Reply:
These are good points. Maybe adding a drop-shadow to the pressed button would help with visual indication of state? But ultimately, some "Thanks for your feedback" text would be a good follow-up.

For button text, there's a visually-hidden mixin we could use to provide screen-readers the text they need. And adding aria-hidden=true on the SVGs would remove the markup from the accessibility tree. (If you want to go down the rabbit hole a bit, there's a pretty thorough [investigation of other SVG patterns from Deque](https://www.deque.com/blog/creating-accessible-svgs/))

We should also add an `aria-pressed` attribute on the button to indicate selected state to screen readers after click (see A True Toggle Button section in Heydon Pickering's [Inclusive Components](https://inclusive-components.design/toggle-button/) for more info)

Reply:
The `aria-pressed` attributes definitely help here, but there's still no descriptive text that a screen reader can use to make sense of what each button does. For example, when I try it with Voice Over on macOS, each button when focused is read aloud as just "button", so it's not possible to tell which is "yes", and which is "no".

I think we really need to use some localized text for each button, and use an image replacement technique (e.g. using `@image-replaced` in CSS), instead of inlining the SVG in the markup?